ledger/participant-state-index/src/main/scala/com/daml/ledger/participant/state/index/v2/ContractStore.scala

```diff
@@ -39,6 +39,12 @@ trait ContractStore {
   def lookupMaximumLedgerTimeAfterInterpretation(ids: Set[ContractId])(implicit
       loggingContext: LoggingContext
   ): Future[MaximumLedgerTime]
+
+  def lookupContractAfterInterpretation(
```

Comment:
Not sure about this name.  Could be verbosely called `lookupContractWithCreatedAt` which is what it technically does.  As it is not intended to be general purpose for for validation maybe `lookupContractForValidation` ?

Reply:
`lookupContractForValidation` sounds better. Thanks!